parlai/tasks/vqa_v1/build.py

```diff
@@ -11,10 +11,13 @@
 
 def buildImage(opt):
     dpath = os.path.join(opt['datapath'], 'COCO-IMG')
+    version = '1'
 
-    if not build_data.built(dpath, version_string='1'):
+    if not build_data.built(dpath, version_string=version):
         print('[building image data: ' + dpath + ']')
-        build_data.remove_dir(dpath)
+        # if already built but updating version, remove outdated files
```

Comment:
let's actually move this comment into the if block (before calling remove_dir), I think it's standard python style to do it that way (weirdly)

maybe change it to
`# an older version exists, so remove these outdated files`